meetings/2021-06-30.md

```diff
@@ -0,0 +1,125 @@
+The authorization server who has access could look at the grant directly and be able to say whether you have access or not.
+Sort of leapfrogging having acp policies applying data grants.
+
+Elf: If you have an algorithm translating the data grants could even not be required.
+If using data grants, you need to be warry of keeping a single source of truth.
+
+Henry: You're bringing a whole bunch of architecture changes. I think it's a little bit like where are the grants... We'll have to look at it and see whether it's better than WAC.
+There's a whole thing about why we may want ACP and a lot of the assumptions for justifying it might have been wrong.
+We should move on to use cases and there, the precise explanation of how things are going, it's difficult to say just by listening to the argument.
+
+Elf: I'm a little worried that the use cases started as WAC UCRs. But Henry, you're right we should compare the solutions. I think it's important to keep things simple from a application development perspective.
+It's a big change of assumptions to have any application have to understand control rules, because I don't think anything that isn't dedicated shouldn't have to care about that.
+
+Henry: I see an authz agent which is on the server and the one on the client. Because the use cases are too close, (two servers owned by the same person), if you just had a server with access it knows everything.
+That's part of what I address in my pull request which is next.
+
+### PR 232 on PR 231: Initial use case for consent workflow
+
+Henry: You have distinction between guards on the server and clients. There should be no entities that has overview of all that, but there's one client that has access to all your verifiable credentials, all your claims and that lives on your phone or computer. The wallet agent is guarding/authorising an app to receive a token. It can decide what apps are allowed to do.
+I think you can even use the same vocabulary.
+Self sovereign identity point of view.
+
+Elf: I think it's useful to think of things in terms of GNAP spec. Resource owner and requesting party authz agents have different responsibilities.
+One is about sharing access with others as a resource owner.
+The other about letting requesting parties doing exactly what they're allowed to do.
+
+Henry: So we have 2 authz agents. You can always delegate. If you delegate too much, you lose sovereignty. I think distinguishing the client/resource authz agents will help.
+The client authenticates and knows when it wants to.
+The server does the reverse.
+
+Elf: That's what we've been discussing with UMA and claims gathering.
+Specific claims gathering can be automated.
+
+Henry: Perhaps we can commit these and try out and find out in details what UMA is.
+We believe in linked data because it's the web architecture friendly way of doing things.
```

Comment:
```suggestion
Henry: Perhaps we can commit these, and try out and find out in detail what UMA is.
We believe in linked data, because it's the web-architecture–friendly way of doing things.
```